sdk/identity/azure-identity/src/main/java/com/azure/identity/InteractiveBrowserCredentialBuilder.java

```diff
@@ -14,7 +14,7 @@
  * @see InteractiveBrowserCredential
  */
 public class InteractiveBrowserCredentialBuilder extends AadCredentialBuilderBase<InteractiveBrowserCredentialBuilder> {
-    private int port;
+    private Integer port;
```

Comment:
Changes from primitive to class.
Primitive gives port a default value of zero.
This is a required parameter and its value should come from user input.

Reply:
It's a required value so the value would never be the default though?

Reply:
That's what my change fixed.
previously port was a primitive, and that had a default value of 0 and consequently builder would never complain in cases when user didn't supply the port as the default value would be zero.

Reply:
I see. Make sense